docs/kb/developer-resources/running-commands-on-the-host-os.md

```diff
@@ -25,7 +25,7 @@ name: "Application"
 properties:
   logo_url: http://www.replicated.com/images/logo.png
   console_title: Application
- 
+
```

Comment:
possibly this one is a bad example. i went through this quickly. more concerned about the ones in block text